examples/simulation/config.yaml

```diff
@@ -55,7 +55,7 @@ train_data_loader:
   batch_size: 12
```

Comment:
Shall we curate a 10hr training set for notebook example?

Reply:
I don't think it's required right now..maybe we can refactor this later to include many changes